crates/telio-dns/src/nameserver.rs

```diff
@@ -53,15 +53,17 @@ pub trait NameServer {
 pub struct LocalNameServer {
```

Comment:
As `nameserver.rs` already have IPv6 support in prod, lets maybe skip the feature flag here.

Reply:
Soo, generally I can remove changes from the whole `telio-dns` crate?

Reply:
As Far As I can see - yes